src/main/kotlin/mate/academy/service/AreaOfFigureSupplier.kt

```diff
@@ -0,0 +1,5 @@
+package mate.academy.service
+
+interface AreaOfFigureSupplier {

```

Comment:
could be implemented by Figure class to avoid passing into each Figure class child.
NOTE! In this case Figure class should be abstract